babel.config.js

```diff
@@ -0,0 +1,9 @@
+const { buildEmberPlugins } = require('ember-cli-babel');
+
+module.exports = function (api) {
+  api.cache(true);
+
+  return {
+    plugins: [...buildEmberPlugins(__dirname)],
+  };
+};
```

Comment:
I'd be in favor of `babelrc.js` actually to keep it in line with `eslintrc.js` etc.

Reply:
@marcoow updated!